src/Command/CommandBase.php

```diff
@@ -261,8 +261,8 @@ public function checkAndPromptTelemetryPreference(): void {
     if (!isset($send_telemetry) && $this->input->isInteractive()) {
       $this->output->writeln('We strive to give you the best tools for development.');
       $this->output->writeln('You can really help us improve by sharing anonymous performance and usage data.');
-      $question = new ConfirmationQuestion('<question>Would you like to share anonymous performance usage and data?</question> ', TRUE);
-      $pref = $this->questionHelper->ask($this->input, $this->output, $question);
+      $style = new SymfonyStyle($this->input, $this->output);
```

Comment:
This looks good, but I'm just confused by Symfony here. If this is the right way to do things it seems like we should change our approach to output in a few places.

Looking at the [Symfony examples](https://symfony.com/doc/current/console/style.html), it seems like we should be naming the `$style` variable `$io` here:

```
        $io = new SymfonyStyle($input, $output);
        $io->title('Lorem Ipsum Dolor Sit Amet');
```

Reply:
Yeah I looked into it more and it does seem like we should audit our code to see where else we can use SymfonyStyle in order to reduce code repetition and improve usability.

Reply:
I chose `$style` based on how [Symfony itself uses the style component](https://github.com/symfony/console/blob/3b597067ee2b07ef6fbbebfe2afa0e88fae9a8a7/Application.php#L268), but I'm not picky on names.